component-library/components/generic/icon/icon.bookshop.yml

```diff
@@ -0,0 +1,613 @@
+# Metadata about this component, to be used in the CMS
+spec:
+  structures:
+    - content_blocks
+  label: "Icon"
+  description: Icons
+  icon: bolt
+  tags: []
+
+# Defines the structure of this component, as well as the default values
+blueprint:
+  svg_icon:
+  big_icon: true
```

Comment:
This parameter is not in the ticket description. May we add this to the ticket description?

Reply:
Yes. I will add it in the ticket.